cmake/download_roc.cmake

```diff
@@ -0,0 +1,57 @@
+include(ExternalProject)
+
+if(DEFINED ENV{CI})
+  set(USE_LOGFILES OFF)
+else()
+  set(USE_LOGFILES ON)
+endif()
+
+set(SCONS_CMD
+  scons
+  --prefix=${CMAKE_CURRENT_BINARY_DIR}/roc-prefix
+  --build-3rdparty=all
+  --enable-static
+  --disable-shared
+  --disable-tools
+  --disable-sox
+  --disable-openssl
+  --disable-libunwind
+  --disable-pulseaudio
+  #--host=${CMAKE_CXX_COMPILER_TARGET}
+  #"CC=${CMAKE_C_COMPILER}"
+  #"CCLD=${CMAKE_C_COMPILER}"
+  #"CXX=${CMAKE_CXX_COMPILER}"
+  #"CXXLD=${CMAKE_CXX_COMPILER}"
+  #"AR=${CMAKE_AR}"
+  #"RANLIB=${CMAKE_RANLIB}"
+  #"STRIP=${CMAKE_STRIP}"
```

Comment:
I think we should uncomment it to allow cross-compiling.

Here is the code that detects CMAKE_CXX_COMPILER_TARGET if it's empty: https://github.com/roc-streaming/roc-pulse/blob/main/cmake/setup_toolchain.cmake (starting from `else(TOOLCHAIN_PREFIX)`).

That code also allows using `TOOLCHAIN_PREFIX` option, which I think can be useful for rt-tests too.

Reply:
Uncommented these lines, but I am not sure what needs to be done for cross compilation. Can this be done in a separate PR?

Reply:
Sure thing.

I've created a separate issue for cross-compilation: https://github.com/roc-streaming/rt-tests/issues/11

Feel free to pick it up or ignore it.